tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -640,6 +626,51 @@ def _run(self):
                         pcnt_used=0
                     ).save()
 
+                # Invoke the aggregated sync flow
+                if "provisioner/gluster" in NS.node_context.tags:
+                    params = {}
+                    params['TendrlContext.integration_id'] = \
+                        NS.tendrl_context.integration_id
+                    payload = {
+                        "tags": ["provisioner/gluster"],
+                        "run": "gluster.flows.SyncAggregatedClusterDetails",
+                        "status": "new",
+                        "parameters": params,
+                        "type": "sds"
+                    }
+                    _job_id = str(uuid.uuid4())
+                    Job(
+                        job_id=_job_id,
+                        status="new",
+                        payload=payload
+                    ).save()
+                    Event(
+                        Message(
+                            priority="info",
+                            publisher=NS.publisher_id,
+                            payload={
+                                "message": "Started job %s for aggregated sync"
+                                           "of cluster %s" %
+                                (_job_id, NS.tendrl_context.integration_id)
+                            }
+                        )
+                    )
+                    while True:
+                        gevent.sleep(3)
+                        job_status = NS._int.client.read(
+                            "/queue/%s/status" % _job_id
+                        ).value
+                        if job_status in ["finished", "failed"]:
+                            msg = "Job %s %s" % (_job_id, job_status)
```

Comment:
Change the msg and add "Job(gluster.flows.SyncAggregatedClusterDetails) %s %s "

and add the actual "payload" in log